chainlink-testing-framework/run-tests/mask_values_in_dot_env.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: go run script.go <base64_dot_env_string>")
+		return
+	}
+
+	decodedBytes, err := base64.StdEncoding.DecodeString(os.Args[1])
+	if err != nil {
+		fmt.Println("Error:", err)
+	}
+
+	envVars := parseEnvVars(string(decodedBytes))
+
+	for key, value := range envVars {
+		mustMaskSecret(key, value)
+		mustAddToGithubEnv(key, value)
+	}
+}
+
+func parseEnvVars(envStr string) map[string]string {
+	envVars := make(map[string]string)
+	lines := strings.Split(envStr, "\n")
+	for _, line := range lines {
+		parts := strings.SplitN(line, "=", 2)
+		if len(parts) == 2 {
+			key := parts[0]
+			value := strings.TrimSpace(parts[1]) // Trim spaces from the value
+			value = removeDoubleQuotes(value)
+			value = removeSingleQuotes(value)
+			envVars[key] = value
+		}
+	}
+	return envVars
+}
+
+func mustAddToGithubEnv(key, value string) {
+	command := fmt.Sprintf("echo \"%s=%s\" >> $GITHUB_ENV", key, value)
+	cmd := exec.Command("bash", "-c", command)
+	err := cmd.Run()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to add '%s' to GITHUB_ENV", key)
+		os.Exit(1)
+	}
+}
```

Comment:
I assume this is working after all? :-) wasn't sure if that would be the case when I proposed to do this part programmatically 👌